### PR TITLE
init and first materials crawl

### PR DESCRIPTION
**Pull request by donutdaniel**

### CODE REVIEW

**Comment by donutdaniel** [comment]
included a example input/output file

**Review by Rahul-Khanna — APPROVED** [review]

**Review by Rahul-Khanna — CHANGES_REQUESTED** [review]
so data should be put in the data folder. 

so in the data folder, you could create subfolder "crawler" and then name your two data files:

"example_input", "example_output"

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 clouds
 .ipynb_checkpoints
 __pycache__
+
+# datasets
+dataset_creation/crawler/__pycache__
```
Comment (Rahul-Khanna) [inline .gitignore]: I would change this to: `**/__pycache__/`  and delete the 3rd line

Comment (donutdaniel) [reply .gitignore]: I think line 3 as it is would suffice

**Comment by donutdaniel** [comment]
the example files should be updated as well with the correct material

**Review by Rahul-Khanna — APPROVED** [review]

**Pull request merged**